tutorials/radiant-mlhub-on-demand-training-data.ipynb

```diff
@@ -0,0 +1,1281 @@
+    "        best_l8_match = min_cloud_cover_scene(\n",
+    "            shape(source_item.geometry), signed_l8_items\n",
+    "        )\n",
+    "\n",
+    "        if not best_l8_match:\n",
+    "            print(\n",
+    "                \"No Landsat 8 Item was found on the Planetary \"\n",
+    "                \"Computer matching the query parameters:\"\n",
+    "            )\n",
+    "            print(\n",
+    "                f\"Source Item ID: {source_item.id} \"\n",
+    "                f\"Bbox: {source_bbox}, \"\n",
+    "                f\"Datetime: {source_datetime}\"\n",
+    "            )\n",
+    "            best_l8_match = None\n",
+    "    else:\n",
+    "        print(\n",
+    "            \"No Sentinel-2 source Item was found in the \"\n",
+    "            \"BigEarthNet dataset matching that label item!\"\n",
+    "        )\n",
+    "        source_item = None\n",
+    "    return source_item, best_l8_match"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "69ad7ef1-143e-47f0-b6b9-26c73e5cc65d",
+   "metadata": {},
+   "source": [
+    "Since it is known that the BigEarthNet dataset from MLHub has a 1-to-1 pairing of source and labels, we can safely assume the first source item is the appropriate match for our label."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ab77ef46-b54a-42e7-a780-bc8e8094ec6f",
+   "metadata": {},
+   "source": [
+    "This makes a connection to the Planetary Computer Data Catalog using the API endpoint URL."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "38280716-e130-4de4-9699-2c0bcbfc056d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mspc_catalog = ps_client.open(MSPC_API_URL)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "39f35d02-d2de-4be5-9317-c80214502c88",
+   "metadata": {},
+   "source": [
+    "We will now use the API client with the helper function above to fetch the best Landsat 8 match for the sampled label Item. This will find only the scenes where the label is completely within the scene, and there is minimal cloud cover."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "53d3ab14-86c7-42a8-be96-8156f5e74d64",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "source_item, best_l8_match = get_landsat_8_match(first_label_item)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "53556cf0-375b-4044-9262-d16de707a13d",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "if best_l8_match:\n",
+    "    print(best_l8_match.id)\n",
+    "    print(best_l8_match.bbox)\n",
+    "    print(best_l8_match.properties)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3f3b5d88-7f3c-4e74-8e68-2408ba762b0f",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "explore_search_extent(ItemCollection([best_l8_match]))"
```

Comment:
Can you also plot the s2 chip's bound here?

Reply:
Similar to other comment threads, would it be best to strip this out of the helper function, and just have a few cells of repeated code for exploring the API search results?

Reply:
Clarifying question on this, do you mean the overall bounds of the ItemCollection returned (e.g. minX, minY, maxX, maxY)? Each Item returned from the API search will have it's own bounding box.